service/src/test/java/datawave/microservice/authorization/AuthorizationServiceTest.java

```diff
@@ -44,9 +46,9 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.fail;
 
```

Comment:
Use @ActiveProfiles and @Profile to limit when test-only classes get instantiated (and injected)